Examples_TODO/TODO2.md

```diff
@@ -0,0 +1,20 @@
+# My contribution on TODO.md (@shirosweets)
+
+### Todo
+
+- [ ] \(Optional) Add autor
+- [ ] Add contributors
+
+### In progress
+- [ ] \(Optional) Add dynamic contributions
```

Comment:
instead of (Optional), we can use P0, P1, P2, P3 at the end for priorities. Optional items can be P3.